app/uk/gov/hmrc/gform/core/parsers/FormatParser.scala

```diff
@@ -127,8 +127,6 @@ object FormatParser {
         TextFormat(ShortText.default)
       } | "shortText(" ~ positiveInteger ~ "," ~ positiveInteger ~ ")" ^^ { (_, _, min, _, max, _) =>
         TextFormat(ShortText(min, max))
-      } | "text" ^^ { (_, _) =>
```

Comment:
This is removing little to much.
I think this parser should stay, it should only produce `TextFormat(TextWithRestrictions(0, 100000))`.
Or did you agree with Alex that `"format":"text"` shouldn't be used?
Also removing of `BasicText` needs to propagate to frontend.

Reply:
I haven't agreed anything with Alex. I thought we were supposed to get rid of text. I will map it to TextFormat(TextWithRestrictions(0, 100000)), as ppl are used to that term

Reply:
"text" has been restored and mapped to `TextFormat(TextWithRestrictions(0, 100000))`

Reply:
Will create fontend PR for removal of BasicText